test/unit/detail/format_man_test.cpp

```diff
@@ -17,7 +17,7 @@ struct format_man_test : public ::testing::Test
     int8_t non_list_pos_opt_value{1};
     std::vector<std::string> list_pos_opt_value{};
     std::string my_stdout{};
-    char const * argv[4] = {"./format_man_test --version-check false", "--export-help", "man"};
+    std::array<char const *, 5> argv = {"./format_man_test", "--version-check", "false", "--export-help", "man"};
```

Comment:
```suggestion
    std::array argv{"./format_man_test", "--version-check", "false", "--export-help", "man"};
```

Reply:
Removing `<char const *, 5>` is not possible, we can't auto-deduct fields of non template classes. (Compilation error if I remove it)

Reply:
`static constexpr std::array argv{...};` would work :D
Not sure if we want it, though?